addon/components/md-tabs.js

```diff
@@ -207,11 +227,7 @@ var MdTabs = Ember.Component.extend(Ember.Evented, LayoutRules, RippleMixin, {
             return false;
         }
         var canvasWidth = this.$().prop('clientWidth');
-        Ember.EnumerableUtils.forEach(this.elements.tabs, function(tab) {
-            canvasWidth -= tab.offsetWidth;
-        });
-
-
+        Ember.EnumerableUtils.forEach(this.elements.tabs, tab => canvasWidth -= tab.offsetWidth);
```

Comment:
Would you be opposed to converting it to a full expression with curly braces? My brain has trouble parsing expressions like this when I see them, and I think having braces and a new line makes the code more readable.